### DISCUSSION TITLE
Potential short-term consultancy - Convert new Risk Data Library Standard into JKAN schema

### DISCUSSION THREAD

**Discussion by pzwsk** [timwis/jkan#286] [Q&A]
Hi everyone, 

as part of the development of the Risk Data Library Standard, supported by the World Bank, we would like to use JKAN to demo the value of the metadata standard for data catalogs.

We are therefore looking for someone to help us "translate" the [JSON formatted RDSL data standard](https://docs.riskdatalibrary.org/en/0__2__0/reference/package_schema/) into a customized JKAN schema.

We already have a JKAN site deployed.

This is for a potential short-term consultancy with work to be done in next few weeks and ahead of the June 16-21 Understanding Risk conference in Japan https://understandrisk.org/event/ur24/

Please let me know if interested.

More info below.

- Risk Data Library website https://riskdatalibrary.org/
- Risk Data Library documentation https://docs.riskdatalibrary.org/en/latest/
- GitHub repository https://github.com/GFDRR/rdl-standard
- JKAN site https://jkan.riskdatalibrary.org/

**Comment by lydiascarf** [comment]
Hi Pierre! i'm a maintainer on [OpenDataPhilly](https://github.com/opendataphilly/opendataphilly-jkan), a JKAN site serving the greater Philadelphia region, and i'd love to help out. Look out for an e-mail from me today.
